charts/policy-controller/templates/policy-webhook/role_policy_webhook.yaml

```diff
@@ -39,6 +39,13 @@ rules:
     resourceNames: ["config-image-policies"]
     verbs: ["get", "list", "create", "update", "patch", "watch"]
 
+  # This is needed to create / patch ConfigMap that is created by the reconciler
```

Comment:
This is from the policy-controller repo: https://github.com/sigstore/policy-controller/blob/067e4fdb8c1f74d0f2a4faddf6723654a597faf2/config/200-role.yaml#L38-L43